src/spgrep_modulation/utils.py

```diff
@@ -72,15 +73,19 @@ def qr_unique(a: NDArrayComplex) -> tuple[NDArrayComplex, NDArrayComplex]:
     return q, r
 
 
-def get_commensurate_diagonal_supercell(qpoint: NDArrayFloat, atol: float = 1e-8) -> NDArrayInt:
-    """Return minimum diagonal supercell in which ``qpoint`` is commensurate."""
+def get_commensurate_diagonal_supercell(qpoint: NDArrayFloat, max_size: int = 10) -> NDArrayInt:
```

Comment:
Can you change `max_size` to `max_denominator` here and docstring?
That is to be consistent with other functions for example: https://github.com/phonopy/spgrep-modulation/blob/c347ab984890ad28f9777a0f11f240c962e4e422/src/spgrep_modulation/isotropy.py#L23-L34